rnd/docker-compose.yml

```diff
@@ -24,10 +24,11 @@ services:
     networks:
       - app-network
 
-  rest_server:
+  agent_server:
```

Comment:
`agent_server` is more ambiguous than `rest_server` imo

Reply:
put it back to rest